packages/presentation/lib/screen/home/widgets/home_grid_view.dart

```diff
@@ -52,15 +53,18 @@ class _HomeGridViewState extends State<HomeGridView> {
           : GridView.builder(
               padding: const EdgeInsets.symmetric(horizontal: Dimens.size18),
               itemCount: widget.movieData.length,
-              gridDelegate: const SliverGridDelegateWithFixedCrossAxisCount(
-                childAspectRatio: Dimens.size167 / Dimens.size320,
-                crossAxisCount: crossAxisCountSize,
-                crossAxisSpacing: Dimens.size2,
+              gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
```

Comment:
you can use https://api.flutter.dev/flutter/rendering/SliverGridDelegateWithMaxCrossAxisExtent-class.html

Reply:
used